dev-java/tomcat-native/tomcat-native-1.2.17.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit java-pkg-2 java-ant-2
+
+DESCRIPTION="Allows Tomcat to use certain native resources for better performance"
+HOMEPAGE="https://tomcat.apache.org/"
+SRC_URI="mirror://apache/tomcat/tomcat-connectors/native/${PV}/source/${P}-src.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="static-libs test"
+
+RDEPEND="dev-libs/apr:1=
+	dev-libs/openssl:=
+	>=virtual/jre-1.8"
+
+DEPEND=">=virtual/jdk-1.8
+	test? ( dev-java/ant-junit )"
```

Comment:
*Always* include a SLOT for non-VM Java dependencies.

Reply:
Java packages add new SLOTs more frequently than other packages and things break quickly when that happens, not least because the jars are installed to `/usr/share/foo-${SLOT}/lib`. For 0, the SLOT is not appended.